tests/modules/programs/fish/abbrs.nix

```diff
@@ -0,0 +1,62 @@
+{ config, ... }: {
+  config = {
+    programs.fish = {
+      enable = true;
+
+      shellAbbrs = {
+        l = "less";
+        gco = "git checkout";
+        "-C" = {
+          position = "anywhere";
+          expansion = "--color";
+        };
+        L = {
+          position = "anywhere";
+          setCursor = true;
+          expansion = "% | less";
+        };
+        "!!" = {
+          position = "anywhere";
+          function = "last_histroy_item";
```

Comment:
There's a typo here, not that it matters

Reply:
Thanks, I fixed it.